components/dropdown/dropdown-content-mixin.js

```diff
@@ -244,7 +276,20 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 	}
 
 	close() {
-		this.opened = false;
+		const hide = () => {
+			this._closing = false;
```

Comment:
I've added a state called _closing to allow the fade-out animation to happen. Previously, the dropdowns just disappeared without transitions, so this state allows the dropdown to be "open" but still have a closing animation.